currency.go

```diff
@@ -37,7 +37,8 @@ var currencies = map[string]*Currency{
 	"BOB": {Decimal: ".", Thousand: ",", Code: "BOB", Fraction: 2, Grapheme: "Bs.", Template: "$1"},
 	"BRL": {Decimal: ",", Thousand: ".", Code: "BRL", Fraction: 2, Grapheme: "R$", Template: "$1"},
 	"BSD": {Decimal: ".", Thousand: ",", Code: "BSD", Fraction: 2, Grapheme: "$", Template: "$1"},
-	"BTN": {Decimal: ".", Thousand: ",", Code: "BTN", Fraction: 2, Grapheme: "Nu.", Template: "1$"},
+  "BTC": {Decimal: ".", Thousand: ",", Code: "BTC", Fraction: 2, Grapheme: "\u20bf", Template: "1$"},
```

Comment:
1) BTC should not have 2 fractions, Satoshi has 8 decimals

2) fix tabs with gofmt

Reply:
3. maybe use `₿` instead of `\u20bf`?

(I am not contributor here, so I cannot merge :) just passing through )

Reply:
> 1. BTC should not have 2 fractions, ~Satoshi~ bitcoin has 8 decimals
> 2. fix tabs with gofmt

fractions are fixed (18 for ETH as well). However, gofmt works different with my default setup then this repo was used. My newest commit changed the whole formatting

Reply:
Hm, sorry for the gofmt suggestion :( I didn't know it will screw up everything this way. I guess this repo did not use `gofmt` in the first place.

Reply:
OKay, I will revert this commit and try to adapt the formation somehow that was originally used :)

Reply:
yeah I think switching everything to correct `gofmt` should be a separate PR

Reply:
Finally I could add these two lines without remotting using VIM instead of GoLand :)